swan-lake/resources/featured-scenarios/work-with-llms-using-natural-expressions.md

```diff
@@ -168,94 +192,12 @@ service on new http:Listener(8080) {
 }
 ```
 
-## Configure the LLM
-
-### Configure the default LLM
-
-Provide configuration for the LLM via configurable variables. This configuration will be used in an LLM client across all natural expressions, unless a model is explicitly specified in the natural expression.
-
-You can use your keys and configuration for OpenAI or Azure OpenAI by configuring the `defaultModelConfig` variable in the `ballerina/np` module.
+## Configure the default model
 
-E.g., add the following for Azure OpenAI in the Config.toml file
-
-```toml
-[ballerina.np.defaultModelConfig]
-serviceUrl = "<SERVICE_URL>"
-deploymentId = "<DEPLOYMENT_ID>"
-apiVersion = "<API_VERSION>"
-connectionConfig.auth.apiKey = "<YOUR_API_KEY>"
-```
-
-Alternatively, you can use the default model made available via WSO2 Copilot. Log in to WSO2 Copilot, open up the VS Code command palette (`Ctrl + Shift + P` or `command + shift + P`), and run `Configure Default Model for Natural Functions`. This will add configuration for the default model into the Config.toml file. Please note that this will require VS Code being open in the relevant directory.
+If you are using the default model made available via WSO2 Copilot (i.e., using the `ai:getDefaultModelProvider()` function), log in to WSO2 Copilot, open up the VS Code command palette (`Ctrl + Shift + P` or `command + shift + P`), and run `Configure Default Model`. This will add configuration for the default model into the Config.toml file. Please note that this will require VS Code being open in the relevant directory.
 
 ![Configure the default model](/learn/images/ai_natural_expr_configure_default_model.png)
```

Comment:
This image needs to change too, but depends on the command change also.

Might have to change the command in the content above too.

Reply:
Updated.